lib/her/model/orm.rb

```diff
@@ -174,10 +174,8 @@ def scope(name, code)
             instance_exec(*args, &code)
           end
 
-          # Add the scope method to the Relation class
-          Relation.instance_eval do
-            define_method(name) { |*args| instance_exec(*args, &code) }
-          end
+          # Add the scope method to the default/blank relation
+          scoped.define_singleton_method(name) { |*args| instance_exec(*args, &code) }
```

Comment:
Noticed an issue with child classes no longer inheriting scopes. Can we make sure that's still available?

Here's a failing spec
```ruby
before do
  spawn_model "Foo::User" do
    scope :active, -> { where(status: "active") }
  end

  Foo::Admin = Class.new(Foo::User)
end

it "passes scopes to children" do
  expect(Foo::Admin.scoped).to respond_to(:active)
end
```

Reply:
There was already a spec for scope inheritance but it didn't cover the chained scope case -- updated that to support both.